### PR TITLE
Improvements: Typos and grammatical errors in README.md

### LINKED ISSUE #1080: [📝 Docs]: Typos and grammatical errors in README.md
### What Docs changes are you proposing?

Quick update to the README.md, there are quite a few grammatical errors and typos

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by IyaadArshad**
**What kind of change does this PR introduce?**
- Fixed some typos, grammatical errors and added extra info in places for extra clarity

**Issue Number:**

-  Closes #1080

**If relevant, did you update the documentation?**
- Minor updates to README.md

**Summary**
- Quick fix for some grammatical errors, typos and improvements on some information

<p>*Feel free to check if there's anything I've missed out on*</p>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +20,15 @@
 </p>
 
 # 👋 Welcome to the JSON Schema website
-This repository contains the sources of JSON Schema website:
+This repository contains the source code of JSON Schema NextJS app:
```
Comment (DhairyaMajmudar) [inline README.md:23]: I think website would be an appropriate word here.

```suggestion
This repository contains the source code of JSON Schema website:
```

**Comment by IyaadDev** [comment]
Ready to merge 👍
This pr just goes through the README.md though
There are plenty more of these grammatical issues around the documentation, will look over some of them and post an issue and pr on those too

**Comment by DhairyaMajmudar** [comment]
Wow! Pretty quick, Thanks for the PR @IyaadDev... You can use this PR for pushing other changes too.

**Comment by IyaadDev** [comment]
Later throughout the course of this week hopefully I'll be able to review the rest of the docs to suggest changes,
thanks @DhairyaMajmudar

File: README.md [hunk 2]
```diff
@@ -53,23 +53,24 @@ This repository has the following structure:
...
 Here are some areas where you can contribute to the website:
+
 - Blogs posts
 - Case Studies
 - Design
-- Documentation
+- Documentation improvements
 - Website enhancements
-- Add a new JSON Schema Implementation
+- JSON Schema Implementations
```
Comment (benjagm) [inline README.md:63]: "Add a new JSON Schema Implementation" makes sense here. Please keep it.

**Review by benjagm — CHANGES_REQUESTED** [review]
Thanks for your contribution. I left just 1 single comment.

Good changes!

File: README.md [hunk 3]
```diff
@@ -20,15 +20,15 @@
...
-* It uses Tailwind CSS framework,
-* It's build and deployed with Netlify.
+* Powered by Next.js,
+* Uses the Tailwind CSS framework,
+* Built and deployed with Netlify.
 
 ## Setting up Project
 
-Please refer to the [INSTALLATION.md](./INSTALLATION.md) file for instructions on how to set up the project on your local machine.
+Please refer to [INSTALLATION.md](./INSTALLATION.md) for instructions on how to set up the project on your machine.
```
Comment (gregsdennis) [inline README.md:31]:
```suggestion
Please refer to [INSTALLATION.md](./INSTALLATION.md) for instructions on how to set up the project for local development.
```

File: README.md [hunk 4]
```diff
@@ -53,23 +53,24 @@ This repository has the following structure:
...
 
 You should also check out our [Contributing guidelines](./CONTRIBUTING.md).
 
 ### Contributors
 
 Thanks goes to these wonderful people who contributed to this website:
+<br />
 <a href = "https://github.com/json-schema-org/website/graphs/contributors">
   <img src = "https://contrib.rocks/image?repo=json-schema-org/website"/>
 </a>
```
[the remark this answers is not in the document]
Comment (IyaadDev) [reply README.md:76]: I think this should work?
```md
[![Contributors](https://contrib.rocks/image?repo=json-schema-org/website)](https://github.com/json-schema-org/website/graphs/contributors)
```

Preview:
[Contributors](https://github.com/json-schema-org/website/graphs/contributors)

File: README.md [hunk 5]
```diff
@@ -83,15 +84,16 @@ Thanks goes to these wonderful people who contributed to this website:
 ### Backers
 
 **Thank you to all our backers!**
+
 [![Backers](https://opencollective.com/json-schema/backers.svg)](https://opencollective.com/json-schema/backers.svg?avatarHeight=90)
 
 ## Connect with the JSON Schema Community
 
 <p align="left">
-    <a href="https://json-schema.org/slack" target="blank" style="margin-right: 5px;"><img align="center" src="https://img.icons8.com/color/48/null/slack-new.png" alt="JSON Schema Slack" height="30" width="40" /></a>
```
[the remark this answers is not in the document]
Comment (gregsdennis) [reply README.md:91]: These each have height specified, which can't be put into markdown.

File: README.md (hunk 2 above)
Comment (benjagm) [inline README.md:63]: Please keep "Add a new JSON Schema Implementation"

**Review by benjagm — CHANGES_REQUESTED** [review]
Just left a small comment.

**Comment by IyaadArshad** [comment]
I'll open a separate pull request for the rest of the changes,
that's about it for changes to the README.md

**Comment by IyaadArshad** [comment]
#1080

**Review by benjagm — APPROVED** [review]

**Comment by benjagm** [comment]
Great work!! Thanks!!

**Pull request merged**